sdk/servicebus/Azure.Messaging.ServiceBus/tests/Samples/Sample01_HelloWorld.cs

```diff
@@ -41,10 +42,10 @@ await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitionin
                 ServiceBusReceivedMessage receivedMessage = await receiver.ReceiveAsync();
 
                 // get the message body as a string
-                string body = Encoding.UTF8.GetString(receivedMessage.Body.ToArray());
+                string body = receivedMessage.Body.AsString();
```

Comment:
Well, isn't this a nice improvement?

Reply:
It was even nicer when we had that implicit cast in there.